CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.29.0 30-06-2021
+
+- Update API version to v10 (#547)
+- Disable preserve downstream scheme (#546)
```

Comment:
line 4 change to:
restored the old way of setting scheme header according to upstream connection security level.

Reply:
Done.